frontend/src/components/NavBar/index.tsx

```diff
@@ -1,32 +1,51 @@
 import { useContext } from 'react'
+import { Button, Container, Nav, Navbar } from 'react-bootstrap'
+import { FaSignInAlt, FaUser, FaHome, FaSignOutAlt } from 'react-icons/fa'
 import { Link } from 'react-router-dom'
 
 import { AuthContext } from '../../context/AuthContext'
 import { CanAccess } from '../CanAccess'
 
 export function NavBar () {
-  const { isAuthenticated, user, signOut } = useContext(AuthContext)
+  const { isAuthenticated, signOut } = useContext(AuthContext)
```

Comment:
@BumbleGoose keeping the user allows you to display the user name of the signed in user.